config.go

```diff
@@ -54,13 +54,19 @@ type Config struct {
 			// Whether or not to use SASL authentication when connecting to the broker
 			// (defaults to false).
 			Enable bool
+			// The type of SASL mechanism to enable. Possible values: OAUTHBEARER, PLAIN (defaults to PLAIN)
```

Comment:
Comments should begin with the name of the field: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences